mk/variables.mk

```
@@ -37,7 +37,10 @@ LOAD_DB_CFG_WITH_YQ := y
 endif
 endif
 
-DATABASE_CONTAINER_NAME="postgres-content"
+COMPOSE_PROJECT_NAME ?= cs
+export COMPOSE_PROJECT_NAME
+
+DATABASE_CONTAINER_NAME=$(COMPOSE_PROJECT_NAME)_postgres-content_1
```

Running it by docker in apple m1, it seems that followed a different name pattern; I suggest the below change:
```suggestion
ifeq (podman,$(DOCKER))
DATABASE_CONTAINER_NAME=$(COMPOSE_PROJECT_NAME)_postgres-content_1
else
DATABASE_CONTAINER_NAME=$(COMPOSE_PROJECT_NAME)-postgres-content-1
endif
```